src/lava/magma/runtime/message_infrastructure/test/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+include(FetchContent)
+FetchContent_Declare(
+  googletest
+  GIT_REPOSITORY https://github.com/google/googletest.git
+  GIT_TAG release-1.12.1
+)
+
+# For Windows: Prevent overriding the parent project's compiler/linker settings
+set(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
+FetchContent_MakeAvailable(googletest)
+
+include_directories(../message_infrastructure/csrc)
+
+add_executable(
+  test_multiprocessing
+  test_multiprocessing.cc
+)
+
+target_link_libraries(
+  test_multiprocessing
+  GTest::gtest_main
+  pybind11::embed
+  message_infrastructure
```

Comment:
I build error, fixed by adding "lt" to target_link_libraries

Reply:
Added librt to fix the build issues in the CI as well.
We also need to add exceptions to GoogleTest... it's currently failing linting and security tests